internal/gguf/model.go

```diff
@@ -97,11 +97,3 @@ func (m *Model) Config() (types.Config, error) {
 func (m *Model) Descriptor() (types.Descriptor, error) {
 	return mdpartial.Descriptor(m)
 }
-
-func (m *Model) GGUFPath() (string, error) {
```

Comment:
how the gguf file will be found in model-runner without this?

Reply:
I left this function on `types.Model` interface (the type returned from the client) but removed it from the `types.ModelArtifact` interface. So, a model returned from the store will have a local gguf file accessible but this isn't guaranteed for any artifact we are manipulating (e.g. an aritfact in the registry). In this particular case we _could_ implement this, but we don't actually have a need for it.